### ISSUE TITLE
pytest-bdd unable to recognize -n argument

### ISSUE THREAD

**Issue by Enigmaderockz** [pytest-dev/pytest-xdist#845]
I am using below command to run pytest-bdd tests 

``
pytest-bdd -sv test_test1.py
``
but when i use -n argument to run pytest-bdd tests in parallel, pytest-bdd is not recognizing -n argument

``
pytest-bdd -sv -n 3 test_test1.py
``

**Comment by RonnyPfannschmidt** [comment]
It's not clear what you are asking for, this plugin is for pytest, not for pytest-bdd

**Comment by RonnyPfannschmidt** [comment]
As far as I can tell the docs for pytest-bdd also recommend running pytest, please elaborate what you actually do

**Comment by Enigmaderockz** [comment]
okay. If you know how can I integrate pytest-bdd with pytest-xdist then it would be very helpful. As far as I know, I can see only pytest usage with this xdist package.

it would be great if we can use pytest-xdist with pytest-bdd framework.

**Comment by nicoddemus** [comment]
Closing as duplicate of https://github.com/pytest-dev/pytest-bdd/issues/584, moving the discussion over there.
